PowerShell/ScubaGear/Rego/AADConfig.rego

```diff
@@ -666,25 +673,44 @@ tests contains {
     "RequirementMet": Status
 } if {
     DescriptionString := "global admin(s) found"
-    Conditions := [
-        count(GlobalAdmins) <= 8,
-        count(GlobalAdmins) >= 2
-    ]
-    Status := count(FilterArray(Conditions, false)) == 0
+    Status := IsGlobalAdminCountGood
 }
 #--
 
 # MS.AAD.7.2v1
 #--
 
-# At this time we are unable to test for 7.2v1
+# Save all users that don't have Global Admin role
+NotGlobalAdmins contains User.DisplayName if {
+    some User in input.privileged_users
+    not "Global Administrator" in User.roles
+}
+
+default GetScoreDescription := "All privileged users are Global Admin"
+GetScoreDescription := concat("", ["Least Privilege Score = ", format_int(x,10), "%"]) if {
+    count(NotGlobalAdmins) > 0
+    x := count(GlobalAdmins)/count(NotGlobalAdmins)*100
+}
```

Comment:
One question about this. In the AAD baseline, the implementation tells you how to calculate the secure score, but the instructions over there produces a ratio, not a percentage, as in:
- The result isn't presented with a percent sign over there
- They don't multiple multiply the result by 100 over there
- They tell you that a passing score is 1 or less. Given that we're multiplying by 100 over here, you'd need to have at least 100 non-global admins for every global admin to achieve that. Probably not what we're going for :)

I don't think either formula is necessarily wrong, but the baseline implementation formula and Rego implementation formulas should match. I do have a slight preference for the ratio version over the percent version, as I think that's a bit more understandable.

Reply:
Optional: what if we also bake some guidance of how to interpret the score into the details column? One option: "Requirement failed: Least Privilege Score: 1.5 (should be 1 or less)"

Reply:
Resolved (after great collaboration on slack) all except the final optional recommendation.

Reply:
Added the extra text per suggestion.